Emergence/Contexts/Main Menu/Getting Shows/ShowEmitter.swift

```diff
@@ -11,6 +11,14 @@ protocol ShowEmitter {
     func showAtIndexPath(index: NSIndexPath) -> Show
     func getShows()
     func onUpdate( callback: EmitterUpdateCallback )
+    
+    func isEqualTo(show: ShowEmitter) -> Bool
+}
+
+extension ShowEmitter {
```

Comment:
:+1: 

This could be improved by conforming to the Equatable protocol: http://swiftdoc.org/v2.0/protocol/Equatable/ ( which you're doing more or less already )